### PR TITLE
Improve the RemoteNode.load_configuration() docs

### PR DESCRIPTION
**Pull request by erlend-aasland**
Spell out how it works and link to PdoBase.read()

Moreover, add return annotation to make sure that's included in the
rendered docs.

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Shamelessly based off of https://github.com/christiansandberg/canopen/pull/405#issuecomment-2198255436, in a slightly more concise wording.

**Review by acolomb — APPROVED** [review]

**Comment by erlend-aasland** [comment]
Thanks for the review!
